### PR TITLE
Add action.name and payload to error message

### LINKED ISSUE #168: More information in case of server error 500
Please give more information about Action.name and payload. This could be in logging.debug, preferred, or in Exception.message.

For instance these are the logs I got this morning from an action with nested actions:
backend_1           | DEBUG:urllib3.connectionpool:http://permission:9005 "POST /internal/permission/is_allowed HTTP/1.1" 500 99
backend_1           | DEBUG:openslides_backend.services.permission.adapter:Permission service response with status code 500: Internal Error. Norman, Do not sent it to client: payload 0: no valid meeting_id or id in payload
backend_1           | ERROR:openslides_backend.services.permission.adapter:Permission service sends HTTP 500 with the following content: Internal Error. Norman, Do not sent it to client: payload 0: no valid meeting_id or id in payload.

### PR DESCRIPTION
**Pull request by ostcar**
Fixes #168

### CODE REVIEW

File: pkg/permission/permission.go [hunk 1]
```diff
@@ -61,7 +61,11 @@ func (ps *Permission) IsAllowed(ctx context.Context, action string, userID int,
 	for i, payload := range payloadList {
 		allowed, err := handler.IsAllowed(ctx, userID, payload)
 		if err != nil {
-			return false, fmt.Errorf("payload %d: %w", i, err)
+			bs, err := json.Marshal(payload)
```
Comment (r-peschke) [inline pkg/permission/permission.go]: I'm no go-programmer, but to me it seems you would clear the content of **err**, but in line 68 you use **err** to return the error message from the handler.IsAllowed-method.

Comment (ostcar) [reply pkg/permission/permission.go]: Thank you very mutch. You are right. This is an ugly bug.

Comment (ostcar) [reply pkg/permission/permission.go]: Done. Please have another look

**Review by r-peschke — APPROVED** [review]
LGTM

**Pull request merged**